libsplinter/src/orchestrator/builder.rs

```diff
@@ -20,9 +20,9 @@ use crate::transport::Connection;
 
 use super::runnable::RunnableServiceOrchestrator;
 
-const DEFAULT_INCOMING_CAPACITY: usize = 8;
-const DEFAULT_OUTGOING_CAPACITY: usize = 8;
-const DEFAULT_CHANNEL_CAPACITY: usize = 8;
+const DEFAULT_INCOMING_CAPACITY: usize = 512;
```

Comment:
This commit is independent and could go into a separate PR.

Reply:
moved to https://github.com/Cargill/splinter/pull/1309